samples/XCT.Sample/Pages/Behaviors/CharactersValidationBehaviorPage.xaml

```diff
@@ -1,69 +1,91 @@
-                                              Source={x:Reference Page}}"
-                        SelectedIndex="1" />
+                        ItemsSource="{Binding CharacterTypes, Source={x:Reference Page}}"
+                        SelectedIndex="1"/>
                 <Label Grid.Row="1"
-                       Text="MinimumCharacterCountEntry" />
-                <Entry x:Name="MinimumCharacterCountEntry"
-                       Grid.Row="1"
-                       Grid.Column="1"
+                       Text="MinimumCharacterCountEntry"/>
+                <Entry x:Name="MinimumCharacterCountEntry" Grid.Row="1" Grid.Column="1"
                        Keyboard="Numeric"
-                       Text="2" />
+                       Text="2"/>
                 <Label Grid.Row="2"
-                       Text="MaximumCharacterCountEntry" />
-                <Entry x:Name="MaximumCharacterCountEntry"
-                       Grid.Row="2"
-                       Grid.Column="1"
+                       Text="MaximumCharacterCountEntry"/>
+                <Entry x:Name="MaximumCharacterCountEntry" Grid.Row="2" Grid.Column="1"
                        Keyboard="Numeric"
-                       Text="20" />
+                       Text="20"/>
             </Grid>
         </Frame>
-        <Label Text="Type characters for validation behavior according to the settings you set upon." />
+        <Label Text="Type characters for validation behavior according to the settings you set upon."/>
         <Entry Placeholder="Type characters...">
             <Entry.Behaviors>
-                <xct:CharactersValidationBehavior CharacterType="{Binding SelectedItem,
-                                                                          Source={x:Reference CharacterTypePicker}}"
+                <xct:CharactersValidationBehavior CharacterType="{Binding SelectedItem, Source={x:Reference CharacterTypePicker}}"
                                                   InvalidStyle="{StaticResource InvalidEntryStyle}"
-                                                  MaximumCharacterCount="{Binding Text,
-                                                                                  Source={x:Reference MaximumCharacterCountEntry}}"
-                                                  MinimumCharacterCount="{Binding Text,
-                                                                                  Source={x:Reference MinimumCharacterCountEntry}}" />
+                                                  ValidStyle="{StaticResource ValidEntryStyle}"
+                                                  MaximumCharacterCount="{Binding Text, Source={x:Reference MaximumCharacterCountEntry}}"
+                                                  MinimumCharacterCount="{Binding Text, Source={x:Reference MinimumCharacterCountEntry}}"/>
+            </Entry.Behaviors>
+        </Entry>
+
+        <Label Text="The same but stying handeled by visual state manager."/>
```

Comment:
```suggestion
        <Label Text="The same but styling handled by visual state manager."/>
```

Reply:
Thanks, corrected it but didn't push, and I blame a next cherry-pick :)